files/jquery.smoothstate/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "npm",
+  "name": "smoothstate",
+  "repo": "miguel-perez/smoothState.js",
+  "files": {
+    "include": ["jquery.smoothState.min.js", "src/jquery.smoothState.js"]
```

Comment:
I dont think this is needed in production src/jquery.smoothState.js
We can remove it